### ISSUE TITLE
[Content] Warning: Unknown prop `tone` on <div> tag

### ISSUE THREAD

**Issue by stam** [CodeYellowBV/re-cy-cle#38]
In TMS I get a console error in `container/Order/Edit` for the tone="primary" on Content.
It does change the color

```
proxyConsole.js:56 Warning: Unknown prop `tone` on <div> tag. Remove this prop from the element. For details, see https://fb.me/react-unknown-prop
    in div (created by Scrollbars)
    in Scrollbars (created by Content__StyledScrollbars)
    in Content__StyledScrollbars (created by Content$1)
    in Content$1 (at Edit.js:175)
    in div (created by ContentContainer)
    in ContentContainer (at Edit.js:151)
    in form (created by Form__StyledForm)
    in Form__StyledForm (created by Form)
    in Form (at Edit.js:150)
    in div (created by Body)
    in Body (at Edit.js:149)
    in OrderEdit (at OrderFullEdit.js:240)
    in OrderEditScreen (at OrderEdit.js:20)
    in OrderEditScreen (at Router.js:47)
    in Route (at Router.js:96)
    in Switch (at Router.js:60)
    in Router (at App.js:45)
    in div (created by Body)
    in Body (at App.js:56)
    in div (created by AppContainer)
    in AppContainer (at App.js:54)
    in App (created by Route)
    in Route (created by withRouter(App))
    in withRouter(App) (at index.js:22)
    in Router (created by BrowserRouter)
    in BrowserRouter (at index.js:21)
    in ThemeProvider (created by ReCyCleTheme)
    in ReCyCleTheme (at index.js:20)
```

**Comment by SpaceK33z** [comment]
I fixed this already in f09134a5136db78ba5fc22a9673f0a318cb1af4b, pls use latest version

**Comment by stam** [comment]
Yes I upgraded from 0.2.11 to 0.2.15 and the warning disappeared
